lib/rex/post/mssql/ui/console.rb

```diff
@@ -58,7 +58,7 @@ def initialize(session, opts={})
 
           # @return [String]
           def database_name
-            session.client.mssql_query('SELECT DB_NAME();')[:rows][0][0]
+            session.client.query('SELECT DB_NAME();')[:rows][0][0]
```

Comment:
Not a blocker; I thought we'd cache this stale value as a default - and then in a separate PR do better street smarts for keeping it up to date